app/src/components/Transcribe.tsx

```diff
@@ -3,34 +3,59 @@ import { Button } from './Controls';
 import { useDispatch, useSelector } from 'react-redux';
 import { abortTranscription, startTranscription } from '../state/transcribe';
 import { TitleBar } from './TitleBar';
-import { AppContainer, CenterColumn } from './Util';
+import { AppContainer, MainCenterColumn } from './Util';
 import { RootState } from '../state';
 import styled from 'styled-components';
+import { openSettings } from '../state/nav';
+import { useState } from 'react';
 
-const FormRow = styled.div`
-  display: flex;
-  flex-direction: row;
-  padding: 10px;
-  font-size: 18px;
+const Form = styled.div`
+  padding: 20px;
+  margin-bottom: 50px;
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-gap: 20px;
 `;
 
 export function TranscribePage(): JSX.Element {
   const dispatch = useDispatch();
   const file = useSelector((state: RootState) => state.transcribe.file) || '';
+  const models = useSelector((state: RootState) =>
+    Object.values(state.models.downloaded).flatMap((x) => x)
+  );
+
+  const [selectedModel, setSelectedModel] = useState(0);
 
   return (
     <AppContainer>
       <TitleBar />
-      <CenterColumn>
-        <FormRow>
-          <p style={{ opacity: 0.5 }}>opened</p>&nbsp;
-          <p> {file.split('/').pop()}</p>
-        </FormRow>
-        <Button primary onClick={() => dispatch(startTranscription())}>
+      <MainCenterColumn>
+        <Form>
+          <span style={{ opacity: 0.5 }}>opened</span>
+          <span> {file.split('/').pop()}</span>
```

Comment:
This might break on windows, right? Let's keep with `path.basename`.

Reply:
whoops that was not an intendet change

Reply:
this is not coming from this pr?

Reply:
Weird, then let's keep it and address it in another PR